src/com/google/javascript/jscomp/parsing/parser/FeatureSet.java

```diff
@@ -199,10 +199,8 @@
     // https://github.com/tc39/proposal-optional-catch-binding
     OPTIONAL_CATCH_BINDING("Optional catch binding", LangVersion.ES2019),
 
-    // Stage 3 proposals likely to be part of ES2020
-    DYNAMIC_IMPORT("Dynamic module import", LangVersion.ES_UNSUPPORTED),
-
     // ES 2020 Stage 4
+    DYNAMIC_IMPORT("Dynamic module import", LangVersion.ES2020),
```

Comment:
Yes, it's officially in ES_2020, but from our point of view, we don't really support it, so we can't put it in that language version yet. If you move this into our definition of ES_2020, you'll also need to modify the definition of the ES_NEXT FeatureSet to subtract this feature, because we cannot safely output it. (It will most likely cause miscellaneous exceptions in optimization passes that don't expect to see this node type.)

The usual procedure for a feature is something like:

1. It moves to ES_NEXT_IN when we can safely accept it in the input and transpile down to at least ES_NEXT level output (or just fail with an error because we cannot transpile it, since that is "safe" behavior).
2. It moves to ES_NEXT when we can safely carry the feature all the way through compilation and output it.

Reply:
@brad4d Based off the call Thursday, I left dynamic import in the ES_2020 category, but added a warning when any usage is encountered.